modules/nf-httpfs/src/main/nextflow/file/http/LatchPath.groovy

```diff
@@ -49,6 +49,10 @@ class LatchPath extends XPath {
         return new URI("latch", fs.domain, path.toString(), null, null)
     }
 
+    String toUriString() {
```

Comment:
pretty sure this is fine but make sure to test cases like 

```
Path lp = LatchPath()
lp.toURIString()
```

and others (i.e. make sure the method resolution never does any fucky things like infinite recursion)